keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -1782,6 +1782,15 @@ Alpha:
 - Adequate documentation for the changes
 - Minimum viable test cases mentioned in Test Plan section
 
+Beta:
+
+- Supports concurrency limiting upon long-running requests
+- Debug endpoint dumping fine-grained states of the queues for priority-levels
+- Allow constant concurrency shares in the priority-level API model
```

Comment:
We already have constant shares. I think you mean constant concurrency / relative shares?

I'm not sure I'd block beta over this.

Reply:
yeah, updated the word to "constant concurrency/relative shares"